src/ripple/app/misc/SHAMapStoreImp.cpp

```diff
@@ -282,7 +286,8 @@ SHAMapStoreImp::run()
     while (1)
     {
         healthy_ = true;
-        validatedLedger_.reset();
+        validatedLedger.reset();
```

Comment:
I like that, nice job. One nit, get rid of the parens after the lambda capture. Also, nice call getting rid of the `mutable`. I was thinking that `newLedger_` was `const` without the mutable, and we wouldn't move from a `const` var. But it works fine without the `mutable`.